pkg/util/fs_util.go

```diff
@@ -261,6 +261,10 @@ func childDirInIgnoreList(path string) bool {
 	return false
 }
 
+func UnTar(r io.Reader, dest string) ([]string, error) {
+	return unTar(r, dest)
```

Comment:
Should we just export `UnTar` instead of exporting this wrapper with the same signature?

Reply:
Yeah, let's do that!